src/lcmap_client/data/surface_reflectance.py

```diff
@@ -4,10 +4,11 @@
 import numpy.ma as ma
 
 from lcmap_client.data import url
+from lcmap_client.util import proj_point_to_tile_point, transform_matrix
```

Comment:
When names start getting like this: ``proj_point_to_tile_point`` it might be time to create a new module (since that's pretty unreadable).

Maybe create ``lcmap_client.projection`` and then have a function ``to_tile_point``. Or something.